pkg/cmd/bootstrap.go

```diff
@@ -235,7 +234,12 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client, cm
 		io.OutputPath = filepath.Join(".", repoName)
 	}
 	io.OutputPath, io.Overwrite = ui.VerifyOutputPath(io.OutputPath, io.Overwrite, outputPathOverridden, promptForAll)
-
+	if !io.Overwrite && io.PushToGit {
+		exists := ui.VerifyGitPath(io.OutputPath)
+		if exists {
+			return fmt.Errorf("the .git folder in output path %s already exists. Delete or rename the .git folder and try again", io.OutputPath)
+		}
+	}
 	if !io.Overwrite {
 		exists := ui.VerifySecretsPath(io.OutputPath)
```

Comment:
move "if io.PushToGit {}" to here.

Reply:
@wtam2018 Thanks for the review. Updated as you mentioned.